index.d.ts

```diff
@@ -0,0 +1,12 @@
+interface Matcher {
+  tag?: string;
+  attrs: {
+    id?: string;
+    class?: RegExp;
+    [attr: string]: any;
+  };
+}
+
+declare function createMatcher(matcher: string | string[]): Matcher | Matcher[];
+
+export default createMatcher;
```

Comment:
there is already a part of types and declarations
https://github.com/posthtml/posthtml-parser/blob/0db4129ec0be89d3278149a4f8140a0c97facd6e/src/index.ts#L16-L28

Reply:
> What's the requested change? To re-use the `NodeTag`?

yep



> One thing to note is that the [`Attributes` type](https://github.com/posthtml/posthtml-parser/blob/0db4129ec0be89d3278149a4f8140a0c97facd6e/src/index.ts#L17) doesn't have `RegExp` as a possible value.

you can always extend the base type